gcc/rust/resolve/rust-default-resolver.cc

```diff
@@ -645,16 +645,24 @@ DefaultResolver::visit (AST::TraitObjectTypeOneBound &)
 {}
 
 void
-DefaultResolver::visit (AST::TupleType &)
-{}
+DefaultResolver::visit (AST::TupleType &type)
```

Comment:
If those types simply require a visit of their subcomponent we might simply remove those functions definition and make use of their parent DefaultVisitor::visit instead of overloading everything with empty function. Unless I'm missing anything ?

Reply:
Alright, yeah, that works a lot better